workshops/erc20-mts/scripts/cli-q.sh

```
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+source .local.env
+
+injectived q $* --node $INJ_URL
```

Shellcheck (0.10.0)

[warning] Use "$@" (with quotes) to prevent whitespace problems.

(SC2048)